src/lib/stepper/stepper-horizontal.html

```diff
@@ -0,0 +1,30 @@
+<div class="mat-horizontal-stepper-header-container">
+  <ng-container *ngFor="let step of _steps; let i = index; let isLast = last">
+    <md-step-header  class="mat-horizontal-stepper-header"
+                     (click)="step.select()"
+                     (keydown)="_onKeydown($event)"
+                     [tabIndex]="_focusIndex == i ? 0 : -1"
```

Comment:
`==` should be `===`. Same goes for the `attr.aria-selected` and `active` bindings.